images/rclone/Dockerfile

```diff
@@ -1,11 +1,18 @@
 FROM golang:1.10-alpine3.7 AS build
 
-COPY rclone /go/src/github.com/ncw/rclone/
+ENV RCLONE_VERSION "v1.45"
+
+WORKDIR "/go/src/github.com/ncw/"
+RUN wget https://github.com/ncw/rclone/releases/download/${RCLONE_VERSION}/rclone-${RCLONE_VERSION}.tar.gz && \
```

Comment:
I'd move the wget to an `ADD` layer. Since you're using a build image anyway, you don't need to clean it up, also `ADD` compares hashes of the previous files that have been downloaded to determine whether a rebuild needs to happen, while `RUN` will just do a lexicographic check of the command which can lead to weird issues when building new images.